src/main/java/com/filmroulette/FilmRouletteController.java

```diff
@@ -30,11 +30,12 @@
     private INowPlayingService nowPlayingService;
 
 
-
+//Endpoint to check for all of the upcoming movies in the MovieDTO array and if there 
+//is an exception return the exception message
```

Comment:
Added javadoc for one of your endpoints, so just make sure all of the methods have appropriate comments